index.html

```diff
@@ -639,15 +639,15 @@
               devices.
             </p>
             <p class="note">
-              User agents SHOULD NOT <a>monitor the list of available remote
-              playback devices</a> when possible, to satisfy the <a href=
+              The user agent is expected not to <a>monitor the list of available
+              remote playback devices</a> when possible, to satisfy the <a href=
               "https://github.com/w3c/remote-playback/blob/gh-pages/use-cases.md#power-saving-friendly">
               power saving non-functional requirement</a>. For example, the
```

Comment:
If power saving is a non-functional requirement, should this be normative?  Or can specs say nothing at all about implementation needed to meet non-functional requirements?

Reply:
I wondered about it as well. I used an informative form in the end because I was trying to think of it in terms of tests. There is no easy way to tell whether the user agent has stopped monitoring the list of available remote playback devices (or is there?), and normative statements that cannot be automatically or manually tested are usually frowned upon.

Reply:
It sounds like non-functional requirements are not in scope: in general, they are statements about the implementation and not directly testable through the Web platform.   So fine to leave this as a non-normative statement.